kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/KieServicesConfigurationImpl.java

```diff
@@ -121,10 +121,24 @@ public KieServicesConfigurationImpl(String url, CredentialsProvider credentialsP
      */
     public KieServicesConfigurationImpl(String url, CredentialsProvider credentialsProvider, long timeout) {
         this.transport = Transport.REST;
-
         this.serverUrl = url;
         this.timeoutInMillisecs = timeout;
         this.credentialsProvider = credentialsProvider;
+        //        this.useSsl = autoDetectUseSSL(url);
+    }
+
+    public boolean autoDetectUseSSL(String url) {
```

Comment:
not sure I follow this method... as the useSsl seems to be used by JMS layer and now we just check the http endpoints... 

https://github.com/kiegroup/droolsjbpm-integration/blob/bc0380c7785501b53943a81f5421d10a4ecc1f3d/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/KieServicesConfigurationImpl.java#L63

could you please explain a bit what are we trying to do here?